### PR TITLE
Several small bug fixes for Mempool bonus guide

### LINKED ISSUE #918: [BUG] Bonus guide: Mempool - 2GB memory not enough for installation
### Describe the bug
Installation fails on RPI4-2GB with 2GB swapfile. Problem rectified with 4 GB swapfile (max allowed on the 2GB version)

### Location in guide
[Link to the corresponding page and section on <https://raspibolt.org>](https://raspibolt.org/bonus/bitcoin/mempool.html)

### Expected behavior
Mempool Installs

### Environment
 - Hardware: Raspberry Pi 4 - 2GB, OS on SD, Data on 1TB SSD
 - Operating system: Rasperry Pi OS 64 bit
 - Version: Debian GNU/Linux 11 (bullseye)

### Additional context
Failure occurred during 'npm run build'.
It should also be noted that there was system instability when trying to operate the system with no swapfile after mempool.space was installed. A 2GB max swapfile seems to have fixed this.

### LINKED ISSUE #965: [BUG] mempool metadata leakage
### Describe the bug
Mempool installation in its current form is leaking information by using the default server for price checks. This should be routed over tor. by adding the following to the mempool config:

    "SOCKS5PROXY": {
        "ENABLED": true,
        "HOST": "127.0.0.1",
        "PORT": 9050
      },
      "PRICE_DATA_SERVER": {
        "TOR_URL": "http://wizpriceje6q5tdrxkyiazsgu7irquiqjy2dptezqhrtu7l2qelqktid.onion/getAllMarketPrices"
      }

### Location in guide
[Link to the corresponding page and section on <https://raspibolt.org>
](https://raspibolt.org/guide/bonus/bitcoin/mempool.html)

### Expected behavior
Pricing data calls should be routed over tor.

### LINKED ISSUE #966: [BUG]  Incorrect command in mempool space install
### Describe the bug
Chown command is executed, but it should be chmod

sudo chown 600 /home/mempool/mempool/backend/mempool-config.json

should be 

sudo chmod 600 /home/mempool/mempool/backend/mempool-config.json

### Location in guide

[https://raspibolt.org/guide/bonus/bitcoin/mempool.html](https://raspibolt.org/guide/bonus/bitcoin/mempool.html)

### PR DESCRIPTION
**Pull request by VajraOfIndra**
#### What

Several bug fixes for the Mempool bonus guide:
- Fixes #918 : states the requirement of running Mempool with a minimum of 4 GB of RAM
- Fixes #965 : add a Tor proxy in Mempool config to fetch prices privately
- Fixes #966 : fixed command typo: chown -> chmod 
- a couple of minor wording changes for improved clarity

#### Scope

- [ ] significant change to core configuration
- [ ] independent bonus guide
- [x] simple bug fix

### CODE REVIEW

File: guide/bonus/bitcoin/mempool.md [hunk 1]
```diff
@@ -179,6 +179,14 @@ For improved security, we create the new user "mempool" that will run the Mempoo
       "USERNAME": "mempool",
       "PASSWORD": "mempool",
       "DATABASE": "mempool"
+    },
+    "SOCKS5PROXY": {
```
Comment (RooSoft) [inline guide/bonus/bitcoin/mempool.md:183]: Can you elaborate on what this does and where does it come from?

**Review by RooSoft — CHANGES_REQUESTED** [review]
All makes sense, just want to know more about the SOCKS5PROXY setting...

File: guide/bonus/bitcoin/mempool.md (hunk 1 above)
Comment (VajraOfIndra) [reply guide/bonus/bitcoin/mempool.md:183]: This was proposed by @visskiss in #965 

The default settings can be found here in the source repo: https://github.com/mempool/mempool/blob/bdfe31c60115d4db1d29f78aa4ee63dd8e3efe71/backend/mempool-config.sample.json
```
  "SOCKS5PROXY": {
    "ENABLED": false,
    "HOST": "127.0.0.1",
    "PORT": 9050,
    "USERNAME": "",
    "PASSWORD": ""
  },
```

Do we need to modify the Tor config to use this?
torrc states that :
```
## Tor opens a socks proxy on port 9050 by default -- even if you don't
## configure one below. [...]
#SocksPort 9050 # Default: Bind to localhost:9050 for local connections.
```

Also, I didn't set up any username and password, but we might need them, I'm not sure....

Comment (RooSoft) [reply guide/bonus/bitcoin/mempool.md:183]: It is working, I have no idea how to make sure the leak is gone, though...

**Review by RooSoft — APPROVED** [review]
All those settings work, just unsure how to check if privacy leaks are completely resolved...

**Comment by visskiss** [comment]
Just thought I'd chime in here. The settings for Tor don't need to be changed because, as you pointed out, Tor provides a default proxy. As for the mempool settings, they actually won't have any effect in the current release (2.3.1) and whoever is installing will need to install from Master branch for these settings to work. If you install from master you will find that the prices are no longer leaked over clearnet. You will also find that another leak that I found in mempool (a file containing pool data) has also been resolved. When 2.4.0 is released, it should include these changes.

**Comment by RooSoft** [comment]
> If you install from master you will find that the prices are no longer leaked over clearnet

How can you test this?

**Pull request merged**